packages/sdk/python/human-protocol-sdk/human_protocol_sdk/escrow/escrow_client.py

```diff
@@ -348,6 +362,7 @@ def create_and_setup_escrow(
         trusted_handlers: List[str],
         job_requester_id: str,
         escrow_config: EscrowConfig,
+        amount: Decimal,
```

Comment:
We should rename this function or even consider removing it since it involves a transfer.

Reply:
I don't know how often we use this function. If you think we can remove this, I'll delete this one.